cmd/derper/derper.go

```diff
@@ -57,6 +57,11 @@ var (
 
 	acceptConnLimit = flag.Float64("accept-connection-limit", math.Inf(+1), "rate limit for accepting new connection")
 	acceptConnBurst = flag.Int("accept-connection-burst", math.MaxInt, "burst limit for accepting new connection")
+
+	egressInterface = flag.String("egress-interface", "", "the interface to monitor for automatic ratelimit tuning")
```

Comment:
Why do we need to monitor specific interfaces? I'd have expected us to measure all non-mesh traffic directly.